Makefile

```diff
@@ -21,9 +21,18 @@
 export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT ?=60s
 export KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT ?=60s
 
+# This option is for running docker manifest command
+export DOCKER_CLI_EXPERIMENTAL := enabled
+
 # Image URL to use all building/pushing image targets
-export CONTROLLER_IMG ?= gcr.io/k8s-cluster-api/cluster-api-controller:latest
-export EXAMPLE_PROVIDER_IMG ?= gcr.io/k8s-cluster-api/example-provider-controller:latest
+REGISTRY ?= gcr.io/k8s-cluster-api
+CONTROLLER_IMG ?= $(REGISTRY)/cluster-api-controller
+EXAMPLE_PROVIDER_IMG ?= $(REGISTRY)/example-provider-controller
+
+TAG ?= latest
+
+ARCH?=amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
```

Comment:
I'm wondering if we should bother with 32-bit `arm` here and instead focus on production-grade hardware.

Reply:
I just copy/pasted all the supported platform list from k8s, I can remove arm from this list.